src/main/resources/static/js/login.js

```diff
@@ -1,28 +1,13 @@
 jQuery(document).ready(function () {
+    function reroutToSubscritpion() {
+        checkBackendSecured();
+        // if (ldapEnabled == false || (ldapEnabled == true && currentUser != null)) {
+        if (isLdapEnabled() && currentUser != null && currentUser != undefined || !isLdapEnabled()) {
```

Comment:
Please group the logical expression used here. I assume that isLdapEnabled() && currentUser != null && currentUser != undefined  is one group and  !isLdapEnabled() is the second group. If that is the case add parathesis around the second group or make it a variable.

The expression is confusing at the first glance and it is frustrating to require to much attention to understand it.

Reply:
Done, reused code already written for other function.